src/java/org/apache/cassandra/tools/nodetool/Scrub.java

```diff
@@ -17,53 +17,61 @@
  */
 package org.apache.cassandra.tools.nodetool;
 
-import io.airlift.airline.Arguments;
-import io.airlift.airline.Command;
-import io.airlift.airline.Option;
-
 import java.util.ArrayList;
 import java.util.List;
 
 import org.apache.cassandra.tools.NodeProbe;
-import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
 import org.apache.cassandra.tools.StandaloneScrubber;
+import org.apache.cassandra.tools.nodetool.layout.CassandraUsage;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+import static org.apache.cassandra.tools.nodetool.CommandUtils.concatArgs;
 
 @Command(name = "scrub", description = "Scrub (rebuild sstables for) one or more tables")
-public class Scrub extends NodeToolCmd
+public class Scrub extends AbstractCommand
 {
-    @Arguments(usage = "[<keyspace> <tables>...]", description = "The keyspace followed by one or many tables")
+    @CassandraUsage(usage = "[<keyspace> <tables>...]", description = "The keyspace followed by one or many tables")
     private List<String> args = new ArrayList<>();
 
-    @Option(title = "disable_snapshot",
-            name = {"-ns", "--no-snapshot"},
+    @Parameters(index = "0", description = "The keyspace followed by one or many tables", arity = "0..1")
+    private String keyspace;
+
+    @Parameters(index = "1..*", description = "The tables to scrub", arity = "0..1")
```

Comment:
Why do we have arity = "0..1" while multiple tables are supported?

Reply:
Fixed.